electron/static/app.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
I wonder if there's a good way to notify a developer when they change the root index file (`public/index.html`) that there's a good chance they should also update this file?

Reply:
One idea might be to have a process that generates both of these files but there are unfortunately subtle differences between the files. Without ejecting from `create-react-app`, I'm not sure best way to do that.